nrt/data/IEEE57/IEEE57_BasicTestCases/IEEE57_4_NodeFault/IEEE57.jobs

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<!--
+    Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+    simulation tool for power systems.
+-->
+<dyn:jobs xmlns:dyn="http://www.rte-france.com/dynawo">
+  <dyn:job name="IEEE57 - Disconnection of generator connected to node 12">
+    <dyn:solver lib="dynawo_SolverIDA" parFile="solvers.par" parId="2"/>
+    <dyn:modeler compileDir="outputs/compilation">
+      <dyn:network iidmFile="IEEE57.iidm" parFile="IEEE57.par" parId="51"/>
+      <dyn:dynModels dydFile="IEEE57.dyd"/>
+      <dyn:precompiledModels useStandardModels="true"/>
+      <dyn:modelicaModels useStandardModels="true"/>
+    </dyn:modeler>
+    <dyn:simulation startTime="0" stopTime="30"/>
+    <dyn:outputs directory="outputs">
+      <dyn:curves inputFile="IEEE57.crv" exportMode="CSV"/>
+      <dyn:logs>
+        <dyn:appender tag="" file="dynawo.log" lvlFilter="DEBUG"/>
```

Comment:
Please remove COMPILE and MODELER logs and put the lvlFilter of the general log to INFO (to avoid useless large files in nrts)